Sources/Core/X86/HardwareBreakpointManager.cpp

```diff
@@ -216,4 +217,35 @@ ErrorCode HardwareBreakpointManager::isValid(Address const &address,
 
   return super::isValid(address, size, mode);
 }
+
+ErrorCode HardwareBreakpointManager::readDebugRegisters(
```

Comment:
Because then you have to know whether you're dealing with a 32-bit or 64-bit CPUState object in each of the callers, instead of in just this one function